src/effects/native/threebandbiquadeqeffect.cpp

```diff
@@ -0,0 +1,392 @@
+#include <effects/native/threebandbiquadeqeffect.h>
+#include "util/math.h"
+
+namespace {
+
+// The defaults are tweaked to match the Xone:23 EQ
+// but allow 12 dB boost instead of just 6 dB
+static const int kStartupSamplerate = 44100;
+static const double kMinimumFrequency = 10.0;
+static const double kMaximumFrequency = kStartupSamplerate / 2;
+static const double kStartupLoFreq = 50.0;
+static const double kStartupMidFreq = 1100.0;
+static const double kStartupHiFreq = 12000.0;
+static const double kQBoost = 0.3;
+static const double kQKill = 0.9;
+static const double kQKillShelve = 0.4;
+static const double kBoostGain = 12;
+static const double kKillGain = 23;
+
+
+double getCenterFrequency(double low, double high) {
+    double scaleLow = log10(low);
+    double scaleHigh = log10(high);
+
+    double scaleCenter = (scaleHigh - scaleLow) / 2 + scaleLow;
+    return pow(10, scaleCenter);
+}
+
+} // anonymous namesspace
+
+
+// static
+QString ThreeBandBiquadEQEffect::getId() {
+    return "org.mixxx.effects.threebandbiquadeq";
+}
+
+// static
+EffectManifest ThreeBandBiquadEQEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("3-Band Biquad EQ"));
```

Comment:
Could you say a bit more on why you think "3-band" belongs in the effect name instead of "Biquad EQ"? 

Our ISOs are 3-band in a sense because they segment the audio into 3 bands. I'm having trouble understanding why it's significant to have in the name when it's not in the name of the others?